Library/Homebrew/compilers.rb

```diff
@@ -111,6 +110,14 @@ def find_compiler
           version = versions.non_apple_gcc_version(name)
           yield Compiler.new(name, version) if version
         end
+      when :clang
+        # silently substitute clang to clang-omp when openmp is required.
+        version = if @formula.needs_compiler_feature?(:openmp)
+          versions.clang_omp_build_version
```

Comment:
Any suggestion to workaround this problem? Or we are forced to go back the old approach?